src/features/offer/components/OfferBody/OfferBody.native.test.tsx

```diff
@@ -402,7 +402,7 @@ describe('<OfferBody />', () => {
     })
   })
 
-  describe('When wipEnableMultivenueOffer feature flag descativated', () => {
+  describe('When wipEnableMultivenueOffer feature flag desactivated', () => {
```

Comment:
Quitte à corriger :
```suggestion
  describe('When wipEnableMultivenueOffer feature flag deactivated', () => {
```

Ce n'est pas comme en français, ça ne prend pas de `s` (cf https://www.google.com/search?q=deactivated&sourceid=chrome&ie=UTF-8)

Reply:
Merci pour le cours d'anglais ^^